doc/release/0.2.rst

```diff
@@ -8,6 +8,7 @@ Release Date: TBD
 
 What's new in ARMI
 ------------------
+#. The method ``Material.density3`` is now called ``density`` (`PR#1999 <https://github.com/terrapower/armi/pull/1999>`_)
```

Comment:
We should probably add a comment on `density` -> `pseudodensity`, yes?

Reply:
Yeah, I improved the language some.  (Sorry, I'm always trying to keep these notes short for people.)